src/persistence.nim

```diff
@@ -290,7 +290,7 @@ proc readLinks_v1(rr; levels: seq[Level]): Links =
   debug(fmt"  numLinks: {numLinks}")
   checkValueRange(numLinks, "links.numLinks", NumLinksLimits)
 
-  result = initBiTable[Location, Location](nextPowerOfTwo(numLinks))
+  result = initLinks(nextPowerOfTwo(numLinks))
```

Comment:
I would only pre-init for performance intensive code where the re-allocations do actually cause a measurable/observable difference. Otherwise it just adds noise to the code, so unless you can prove this makes a big difference, can you please remove it?